sopel/modules/reload.py

```diff
@@ -63,18 +70,21 @@ def f_update(bot, trigger):
     f_reload(bot, trigger)
 
 
-@sopel.module.nickname_commands("load")
-@sopel.module.priority("low")
-@sopel.module.thread(False)
-@sopel.module.require_admin
+@plugin.nickname_command("load")
+@plugin.priority("low")
+@plugin.thread(False)
+@plugin.require_admin
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def f_load(bot, trigger):
     """Loads a plugin (for use by admins only)."""
     name = trigger.group(2)
     if not name:
-        return bot.reply('Load what?')
+        bot.reply('Load what?')
+        return
 
     if bot.has_plugin(name):
-        return bot.reply('Plugin already loaded, use reload')
+        bot.reply('Plugin already loaded, use reload')
```

Comment:
Ditto what I said on `f_reload()`.

```suggestion
        bot.reply('Plugin already loaded; use the `reload` command.')
```

Also ditto about `help_prefix` if you like.